### DISCUSSION TITLE
How about use `.detaignore`

### DISCUSSION THREAD

**Discussion by aciddust** [deta/cloud-docs#518] [Ideas] [answered]
When running `deta deploy`..

It seems good to specify and exclude things that you don't want to distribute by using `.detaignore`

is way already exist what I said?

**Comment by AyushSehrawat** [comment, marked as answer]
`.detaignore` exists. You can specify the files in it that you don't want to be deployed

**Reply by aciddust** [reply]
I have two project in deploy directory
(e.g. Web API Server, User Web Interface)
so I wanna ditch one of them, and need `.detaignore`

When I opened this issue, already added `detaignore` but error occured.
I thought it didn't actually exist due to the lack of detaignore related content in the guide document.

Anyway, I deployed my application
Many thanks
